atlp-rwanda/Dev-Pulse-BN
# ft(BN)_adding_manager's_id_into_user_table

#### What does this PR do?
- Adding and updating managerId column in the user table
- getting manager's info from average rating table because it containes submitter id
- and i chenged few file ; others were chenged by eslint; like converting double quote to single quote

#### Description of Task to be completed?
- I added `managerId` column in user table;
- and Update managerId column when manager add trainee to his group
- Adding relationship between user table and average rating table inorder to fetch submitter's info
- View List of all users ; will return trainee's data with his manager id
- and view `http://localhost:3000/api/v1/ratings` you will see in manager's data too

#### How should this be manually tested?
- first of all copy and paste ` sequelize db:migrate:undo:all  && sequelize db:migrate && sequelize db:seed:all` into your terminal to reset the database
- run `yarn dev`
- navigate to `api/doc`
- Login as usual
- add trainee to a certain group
- view all trainees' list
-  and view `http://localhost:3000/api/v1/ratings` you will see in manager's data too as shown on photo
- find that one you added in the group you will return manager Id included in trainee's data nformation about you

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[]()
#### Screenshots (if appropriate)
![Screenshot 2022-03-03 160721](https://user-images.githubusercontent.com/41394162/156581937-7fd9d86d-9dff-4d08-8f8f-7e49b0eb9d83.png)


#### Questions:
N/A
